pcdsdevices/cvmi_motion.py

```diff
@@ -6,23 +6,10 @@
 
 from ophyd import Component as Cpt
 
-from .device import Device, GroupDevice
+from .device import GroupDevice
 from .epics_motor import BeckhoffAxis
 from .interface import BaseInterface
-from .signal import PytmcSignal
-
-
-class CvmiLED(Device):
-    """
-    Basic control and readback PVs for LED rings attached to the viewports
-    on the vacuum chamber. These include illumination percentage,
-    power ON/OFF, and a naming string field.
-    """
-
-    # LED variables
-    desc = Cpt(PytmcSignal, ':NAME', io='io', kind='normal', string=True)
-    pct = Cpt(PytmcSignal, ':ILL:PCT', kind='normal', io='io')
-    pwr = Cpt(PytmcSignal, ':PWR', kind='normal', io='i')
+from .light_control import LightControl
 
```

Comment:
Typically this would be considered an API break- but in practice I suspect nobody has imported this name yet. If we were being super pedantic we'd probably want to add a backwards-compatibility line:
```
CvmiLED = LightControl
```

Reply:
Good idea. Can this go anywhere or should it go in the CVMI class?

Reply:
Good idea. Can this go anywhere or should it go in the CVMI class?

Reply:
You'd typically want to put it in the same place that the original class was.

The goal would be to make sure that a user who had code structured like:
`from pcdsdevices.cvmi_motion import CvmiLED` would still have their code work after the rename.

I think we can skip it here because it's not really something anyone has imported on their own, but it could also be added.